src/rust/Cargo.toml

```diff
@@ -36,15 +36,15 @@ members = [
 # assumes you have `rust-lib` in the same directory as `ide`. See:
 # https://github.com/enso-org/rust-lib/blob/main/docs/CONTRIBUTING.md#developing-in-conjunction-with-enso--ide
 [patch.crates-io]
-# enso-automata       = { path = '../../../rust-lib/src/automata'       }
-# enso-data           = { path = '../../../rust-lib/src/data'           }
-# enso-generics       = { path = '../../../rust-lib/src/generics'       }
-# enso-logger         = { path = '../../../rust-lib/src/logger'         }
-# enso-macro-utils    = { path = '../../../rust-lib/src/macro-utils'    }
-# enso-optics         = { path = '../../../rust-lib/src/optics'         }
-# enso-prelude        = { path = '../../../rust-lib/src/prelude'        }
-# enso-shapely        = { path = '../../../rust-lib/src/shapely/impl'   }
-# enso-shapely-macros = { path = '../../../rust-lib/src/shapely/macros' }
+#enso-automata       = { path = '../../../rust-lib/src/automata'       }
+#enso-data           = { path = '../../../rust-lib/src/data'           }
+#enso-generics       = { path = '../../../rust-lib/src/generics'       }
+#enso-logger         = { path = '../../../rust-lib/src/logger'         }
+#enso-macro-utils    = { path = '../../../rust-lib/src/macro-utils'    }
+#enso-optics         = { path = '../../../rust-lib/src/optics'         }
+#enso-prelude        = { path = '../../../rust-lib/src/prelude'        }
+#enso-shapely        = { path = '../../../rust-lib/src/shapely/impl'   }
+#enso-shapely-macros = { path = '../../../rust-lib/src/shapely/macros' }
```

Comment:
rm these changes pls

Reply:
I prefer lack of changes. And for this being commented, see the comment above